boot.php

```diff
@@ -38,7 +38,7 @@
 define ( 'FRIENDICA_CODENAME',     'Asparagus');
 define ( 'FRIENDICA_VERSION',      '3.5.1-dev' );
 define ( 'DFRN_PROTOCOL_VERSION',  '2.23'    );
-define ( 'DB_UPDATE_VERSION',      1206      );
+define ( 'DB_UPDATE_VERSION',      1207      );
```

Comment:
Please be aware that - if my pull request might be accepted - I already used this database version number.